src/indra_cogex/sources/processor.py

```diff
@@ -88,7 +88,13 @@ def dump(self):
 
     def _dump_nodes(self) -> Path:
         sample_path = self.module.join(name="nodes_sample.tsv")
-        nodes = sorted(self.get_nodes(), key=lambda x: (x.db_ns, x.db_id))
+        nodes = sorted(
+            [
+                n
+                for n in tqdm(self.get_nodes(), desc="Node generation", unit_scale=True)
```

Comment:
`sorted()` can take an iterable directly, so no need to do the trivial list comprehension around it

Reply:
I made a commit to update this